proto/viam/app/v1/app.proto

```diff
@@ -840,6 +840,13 @@ enum FragmentErrorType {
   FRAGMENT_ERROR_TYPE_CYCLE_DETECTED = 4;
 }
 
+message FragmentUsage {
+  int32 fragment_id = 1;
+  int64 organizations = 2;
+  int64 parts = 3;
+  int64 machines_in_org = 4;
+}
```

Comment:
what is the full set of stats that we need? the fragment message currently reports `robot_part_count` but the [scope designs show a "machines deployment" count](https://docs.google.com/document/d/1bGj1F9pDIUPJ_NF4kc50eYsyP2PNJUjlGbhzyb_Y6Kk/edit?tab=t.0)?

Reply:
Sorry yeah I included extras in my example proto. I think this is the fields needed for the initial scope. The two existing fields in the existing Fragment proto are renamed and one field added:

message FragmentUsage {
  int64 total_organizations = 1; // (was organization_count)
  int64 total_machines = 2;  // (was robot_part_count)
  int64 current_organization_machines = 3;
}

Reply:
is total machines a different number than robot part count?

Reply:
I think we get to decide, but I'd say "yes, let's finally do machines instead of parts." But we can bail out on that and continue to do parts if writing the logic gets too hairy

Reply:
updated to machines!